python/lsst/pipe/base/config.py

```diff
@@ -29,15 +29,19 @@
 # -------------------------------
 #  Imports of standard modules --
 # -------------------------------
+import os
 from numbers import Number
-from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple, Type, TypeVar
+from typing import TYPE_CHECKING, Any, Dict, Iterable, Optional, Tuple, Type, TypeVar
```

Comment:
```suggestion
from collections.abc import Iterable
from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple, Type, TypeVar
```
You don't need to all the way in terms of modernizing type annotations here, but this much should already be consistent with the rest of the module and only involves the new symbol you've imported.